src/lib/siteConfig.js

```diff
@@ -1,14 +1,28 @@
-export const SITE_URL = 'https://swyxkit.netlify.app';
-export const APPROVED_POSTERS_GH_USERNAME = ['sw-yx'];
-export const GH_USER_REPO = 'sw-yx/swyxkit'; // used for pulling github issues and offering comments
+import { env } from '$env/dynamic/public'
+
+// You can edit the config directly by changing this values
+// or pass it via environment variable
+// all env variables starts with PUBLIC_ except GH_TOKEN
+const PUBLIC_SITE_URL ='https://swyxkit.netlify.app'
+const PUBLIC_GH_USER_REPO ='sw-yx/swyxkit'
+const PUBLIC_SITE_TITLE ='SwyxKit'
+const PUBLIC_SITE_DESCRIPTION ="swyx's default SvelteKit + Tailwind starter"
+const PUBLIC_DEFAULT_OG_IMAGE ='https://user-images.githubusercontent.com/6764957/147861359-3ad9438f-41d1-47c8-aa05-95c7d18497f0.png'
+const PUBLIC_MY_TWITTER_HANDLE ='swyx'
+const PUBLIC_MY_YOUTUBE ='https://youtube.com/swyxTV'
+const PUBLIC_POST_CATEGORIES = 'Blog'
+
+export const SITE_URL = env?.PUBLIC_SITE_URL || PUBLIC_SITE_URL
+export const GH_USER_REPO = env?.PUBLIC_GH_USER_REPO || PUBLIC_GH_USER_REPO
+export const SITE_TITLE = env?.PUBLIC_SITE_TITLE || PUBLIC_SITE_TITLE
+export const SITE_DESCRIPTION = env?.PUBLIC_SITE_DESCRIPTION || PUBLIC_SITE_DESCRIPTION
+export const DEFAULT_OG_IMAGE = env?.PUBLIC_DEFAULT_OG_IMAGE || PUBLIC_DEFAULT_OG_IMAGE
+export const MY_TWITTER_HANDLE = env?.PUBLIC_MY_TWITTER_HANDLE || PUBLIC_MY_TWITTER_HANDLE
+export const MY_YOUTUBE = env?.PUBLIC_MY_YOUTUBE || PUBLIC_MY_YOUTUBE
+export const POST_CATEGORIES = (env?.PUBLIC_POST_CATEGORIES || PUBLIC_POST_CATEGORIES).split(',');
```

Comment:
much better but still feels a bit unnecessarily duplicative. wondering if we should just check in a .env file and tell users to modify? why did we .gitignore the `.env` file in the first PR?

Reply:
sorry i dont really know what i want.. just thinking out loud haha

we are basically optimizing for deployment simplicity and readability of code - this solution is plenty readable already so honestly maybe i am just splitting hairs a bit. tell me if you think im overthinking this

Reply:
Because `.env` should never get pushed to git repo.
`.env` is always gitignored and only exists for local testing as an alternative to `export GITHUB_TOKEN=…` etc

but yeah I just tried to implement the ability to use `env` along with your concerns about complexity.
If this looks too much duplication we can leave it as it is.
I do not dislike at it’s current state either.

Reply:
yeah ok lets leave it for now.. sorry.. i dont mind revisiting this in future but i hate mucking around with env vars in the dashboard if i can help it